v2/go.mod

```diff
@@ -0,0 +1,12 @@
+module go.mongodb.org/atlas/mongodbatlas/v2
```

Comment:
This module is transient. Removing module will not make any difference to the end user due to use of the same path. We only use it to separate dependencies between v1